### PR TITLE
[13.0][ADD] payment_environment

### PR DESCRIPTION
**Pull request by ivantodorovich**
Configure payment acquirers with server_environment

### CODE REVIEW

File: payment_environment/models/payment_acquirer.py
Comment (simahawk) [inline payment_environment/models/payment_acquirer.py:3]: AGPL or LGPL? :)

Comment (simahawk) [reply payment_environment/models/payment_acquirer.py:3]: after checking it seems there's a bit of confusion on licenses. 

The main module says GPL but refers to AGPL link, the tech mixing [says AGPL](https://github.com/OCA/server-env/blob/13.0/server_environment/models/server_env_mixin.py#L2) but seems wrong to me.

Would you mind opening another PR to fix them so that we can fast track it?

LGPL seems the right one for the whole thing. Maybe ping @guewen  on the new PR.

File: payment_environment/models/payment_acquirer.py
Comment (simahawk) [inline payment_environment/models/payment_acquirer.py:12]: you can save a line here tnx to https://github.com/OCA/server-env/pull/75

**Review by simahawk — CHANGES_REQUESTED** [review]

File: payment_environment/models/payment_acquirer.py
Comment (ivantodorovich) [reply payment_environment/models/payment_acquirer.py:3]: PR created here: https://github.com/OCA/server-env/pull/77

As for this, my intention was AGPL, this got carried on from a copy & paste left over.
Fixing it now :sweat_smile:

File: payment_environment/models/payment_acquirer.py
Comment (ivantodorovich) [reply payment_environment/models/payment_acquirer.py:12]: Indeed :) but I'll wait until it's merged

**Comment by simahawk** [comment]
#77 is getting merged

**Comment by ivantodorovich** [comment]
Thanks! PR updated accordignly :)

File: payment_environment/__manifest__.py
Comment (simahawk) [inline payment_environment/__manifest__.py]:
```suggestion
# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
```
? :smile:

File: payment_environment/models/payment_acquirer.py
Comment (simahawk) [inline payment_environment/models/payment_acquirer.py]: same

File: payment_environment/views/payment_acquirer.xml
Comment (simahawk) [inline payment_environment/views/payment_acquirer.xml]: guess what? :D

Comment (ivantodorovich) [reply payment_environment/views/payment_acquirer.xml]: :(

Comment (ivantodorovich) [reply payment_environment/views/payment_acquirer.xml]: done!

**Review by simahawk — COMMENTED** [review]
@ivantodorovich I've just realized that this should probably go to https://github.com/OCA/?q=payment

**Comment by simahawk** [comment]
@ivantodorovich ping :)

**Comment by ivantodorovich** [comment]
>  @ivantodorovich I've just realized that this should probably go to https://github.com/OCA/?q=payment

Should it thought? 🤔 

by the same logic, shouldn't `mail_environment` go to https://github.com/OCA/social; `pos_environment` go to https://github.com/OCA/pos; `carrier_environment` go to https://github.com/OCA/delivery-carrier; etc..

This is kind of a gray area I think 😅

**Comment by gurneyalex** [comment]
I would keep this in OCA/server-env if you start using it, it's nice to have all the deps in 1 single place.

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**
